projects/Mallard/src/helpers/settings/defaults.ts

```diff
@@ -63,7 +63,6 @@ export const defaultSettings: Settings = {
     notificationServiceRegister: __DEV__
         ? notificationServiceRegister.code
         : notificationServiceRegister.prod,
-    zipUrl: apiUrl + 'zips',
```

Comment:
We still need to keep this (just minus `zips`) as the fetch call above does not have a domain as it stands.